src/Messages.js

```diff
@@ -979,5 +979,10 @@ export default defineMessages({
         id: 'redhatDisabledRuleAlertTitle',
         description: 'Red Hat disabled rule alert title',
         defaultMessage: 'Red Hat has proactively disabled certain low risk recommendations'
+    },
+    inventoryIdNotFound: {
+        id: 'inventoryIdNotFound',
+        description: 'Thrown as error when classic id does not correspond to an inventory id',
+        defaultMessage: 'No system found in inventory for the given Insights ID'
```

Comment:
should this say Advisor ID instead ?  🤔

Reply:
I was going off of `/api/inventory/v1/hosts?insights_id=...` but we could change it to that 🤷‍♂️

Reply:
Yeah lets change that, ux might say differently, but Advisor is the product (in the Insights portfolio?), lil more meaningful methinks 🤔